### ISSUE TITLE
Fixed an issue with StyleRegEx which prevents application of font attributes

### ISSUE THREAD

**Issue by tereks** [malcommac/SwiftRichString#58]
Hi. I want to implement such thing:

```swift
let highlightedStyle = StyleRegEx(base: style, pattern: patternText) {
            $0.font = UIFont.systemFont(ofSize: 18, weight: .bold)
            $0.color = UIColor.black
}
```

Currently I cannot get such results

**Comment by malcommac** [comment]
There was an issue with cached values (https://github.com/malcommac/SwiftRichString/commit/13d217e6caef36772c05d31bf1d742011781e6e6#diff-33601a626565362c1b3b88f5c7cd9ca4L112).
Fixed in 2.1.0.
Thanks.

**Comment by malcommac** [comment]
Example text:

```swift
		let patternText = "[A-Z0-9a-z._%+]+@[A-Za-z0-9.]+\\.[A-Za-z]{2,4}"
		let bodyHTML = "daniele hello@danielemargutti.com ciao ciao"

		let baseStyle = Style {
			$0.font = UIFont.boldSystemFont(ofSize: 12)
		}

		let highlightedStyle: StyleRegEx = StyleRegEx(base: baseStyle, pattern: patternText) {
			$0.font = UIFont.systemFont(ofSize: 40, weight: .bold)
			$0.color = UIColor.red
		}!
		self.textView?.attributedText = bodyHTML.set(style: highlightedStyle)

```
